### PR TITLE
[disasters] fix severity prop name for earthquake magnitude

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/config/subject_page_pb2.py [hunk 1]
```diff
@@ -2,9 +2,10 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: subject_page.proto
 """Generated protocol buffer code."""
-from google.protobuf.internal import builder as _builder
```
Comment (shifucun) [inline server/config/subject_page_pb2.py:5]: Curious what protoc version do you use? The format of this file has been changing through the PRs

Comment (beets) [reply server/config/subject_page_pb2.py:5]: i've also been explicitly skipping submitting the file for most prs.

we could add the protos to be generated at build time, and not check it in.. wdyt?

**Review by beets — APPROVED** [review]
thanks for the fix!

we could revert the compiled proto file for this pr..

File: server/config/subject_page_pb2.py (hunk 1 above)
Comment (chejennifer) [reply server/config/subject_page_pb2.py:5]: I'm using 3.19.4 

but yes, will revert this file!

Comment (shifucun) [reply server/config/subject_page_pb2.py:5]: Not checking this in might work, but we need to ensure `protoc` runs in local/webdriver/cloudbuild etc.. For now, I can add a check for protoc version in `run_server.sh`? If it's not desired version, protoc won't run...

Comment (shifucun) [reply server/config/subject_page_pb2.py]: https://github.com/datacommonsorg/website/pull/1945 sent out a fix for this

**Pull request merged**